src/TempAuthAccpStage.ts

```diff
@@ -1,21 +1,13 @@
 import {
-  Environment, Stack, StackProps, Stage, StageProps,
+  Stack, StackProps, Stage,
   aws_ssm as SSM,
   aws_route53 as route53,
 } from 'aws-cdk-lib';
 import { Construct } from 'constructs';
 import { Statics } from './Statics';
 
-export interface TempAuthAccpStageProps extends StageProps {
```

Comment:
Dit is allemaal ook niet meer nodig?

Reply:
Properties waren niet in gebruik. Dit is overgebleven van het kopieren om deze stage aan te maken.